03-devops-overview.Rmd

```diff
@@ -0,0 +1,183 @@
+# DevOps - The Basics
+
+## Background
+
+So far we have only looked at Git in the context of local projects and command line software. This alone doesn't make collaborating on code easy. 
+
+What is needed is a central platform where people can request for changes to be made to the code base and update their code based on changes by others.  
+
+This concept in Git is known as [remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) or remote repositories.
+The following are examples of interactive services that allow you to manage Git projects effectively:
+
+- [GitHub](github.com)
+- [Bitbucket](https://bitbucket.org)
+- [GitLab](gitlab.com/)
+- [Azure DevOps](https://azure.microsoft.com/en-us/services/devops/).
+
+We are using **Azure DevOps** as this is the Department's tool of choice for internal version control. We would use platforms such as GitHub for our more publicly facing projects (such as this book).
+
+To work with internal DevOps Repos you will need an account with Basic access level.
+There is a form to sort out DevOps account levels and access on the IT service portal - search for DevOps.
```

Comment:
Can we link to IT service portal form? Or is that not a good idea in a public facing document?

Reply:
As with the DevOps URL, I think not in a public document